docs/dev/features/netapp-mounting/overview.md

```diff
@@ -30,42 +32,38 @@ This controller is built off of the existing `aaw-kubeflow-profiles-controller`
+- Determine if a bucket at the user requested path exists, if not create it
 
-**Until the above works, you must manually create secrets**
+A TLDR;
 
-## Manually Creating Secrets
-To do this, you need the following information; `namespace`, `S3_ACCESS`, `S3_BUCKET`, `S3_SECRET`, and `S3_URL`. The actual entries of the secret (S3) can be retrieved from the Netapp team, as when users are onboarded someone will send you their ACCESS and SECRET keys. The bucket name would match what the share name is in the request, for example in [this issue](https://jirab.statcan.ca/browse/ZPS-24?focusedId=3011155&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-3011155) the value for `S3_BUCKET` would be `istdim`, as the bucket names must be lowercased (another one to look out for is `_`'s get converted to `-`'s, if unsure ask Roham for the bucket name. 
-For the `S3_URL` this is constant and can view the respective URL from existing secrets
-Do not forget to do encode them when creating the secret, I have a utility that just does `echo -n $1 | base64` that I call so I can copy paste it easily.
+User selects and inputs a path in a filer they want access to via the UI which then creates a configmap. This controller picks up that configmap and checks if a user for that filer exists using the onpremname from the graph api, if not creates and assigns it the correct policy to interact with it. The controller then hashes the user inputted filer path to a bucket name and then we check if that bucket exists, if it does not create it.
+The controller then cleans up and creates or modifies the `existing-shares` configmap which is used by the filer-sidecar-injector.
 
-The naming convention of the secret is **VERY IMPORTANT** it must end in `filer-conn-secret` for everything to work.
-In general, it should look like `{filerName}-{bucketName}-filer-conn-secret` where `filerName` is the name of the filer, for example if you are creating something in the field 9 filer it should be `fld9`. `bucketName` is the same value from above, this is needed in the event that a field filer has multiple buckets needed so we can distinguish and unique the secrets (for example s3bucket).
+For more details on how this controller works, please refer to the [README](https://github.com/StatCan/aaw-kubeflow-profiles-controller/blob/profiles-controller-aaw2.0/ontap-cvo.md)
 
-So a full example secret name could be `fld9-s3bucket-filer-conn-secret`
+### Deployment Checklist
+This is currently built off of the `profiles-controller` so the deployment here is the same as the namespace controller above. Just make sure that you are on the [profiles-controller-aaw2.0 branch](https://github.com/StatCan/aaw-kubeflow-profiles-controller/tree/profiles-controller-aaw2.0)

```

Comment:
Should we mention the charts repo here? Or is that implied knowledge from the profiles-controller deployment?

Reply:
added link to `namespace controller` which also has the same details here